src/main/java/com/code_intelligence/jazzer/instrumentor/Hooks.kt

```diff
@@ -81,12 +82,12 @@ data class Hooks(
                     val hookClass =
                         Class.forName(hookClassName, true, Companion::class.java.classLoader)
                     loadHooks(hookClass).also {
-                        println("INFO: Loaded ${it.size} hooks from $hookClassName")
+                        Log.info("Loaded ${it.size} hooks from $hookClassName")
                     }.map {
                         it to hookClass
                     }
                 } catch (e: ClassNotFoundException) {
-                    println("WARN: Failed to load hooks from $hookClassName:")
+                    Log.warn("Failed to load hooks from $hookClassName:")
```

Comment:
Combine with exception and remove next line.

Reply:
Changed.